requirements.txt

```diff
@@ -1,8 +1,8 @@
-librosa==0.7.0
-numpy==1.17.1
-opencv-contrib-python>=4.2.0.34
-opencv-python==4.1.0.25
-torch==1.1.0
-torchvision==0.3.0
-tqdm==4.45.0
-numba==0.48
+librosa
+numpy
+opencv-contrib-python
+opencv-python
+torch
+torchvision
+tqdm
+numba
```

Comment:
Not sure if its a good idea to remove the version. The newer versions of the packages might contain breaking changes.

Also, have to tested installing the requirements in the new venv. When I last tried, numba==0.48 was the must have.